lib/logger.js

```diff
@@ -358,11 +364,16 @@ Logger.prototype._flush = function(callback) {
  *  Populate short-hand for each supported Log Level
  */
 configs.LOG_LEVELS.forEach(function(level) {
-    var l = level.toLowerCase();
-    Logger.prototype[l] = function(statement, opts) {
-        opts = opts || {};
+    var levelFuncationName = level.toLowerCase();
+    Logger.prototype[levelFuncationName] = function(statement, opts, callback) {
+        if (opts && typeof opts === 'function') {
+            callback = opts;
+            opts = {};
+        }
+        if (!opts) opts = {};
```

Comment:
I prefer `opts = opts || {};` format. If you do have it this way, let's bracket the clause in the `if` - 1 line if statements like this can lead to errors when refactoring.